common/src/main/java/com/wynntils/functions/HorseFunctions.java

```diff
@@ -108,4 +125,21 @@ protected List<String> getAliases() {
             return List.of("h_name");
         }
     }
+
+    public static class HorseLevelTimeFunction extends Function<Double> {
+        @Override
+        public Double getValue(FunctionArguments arguments) {
+            Optional<HorseItem> horse = Models.Horse.getHorse();
+            if (horse.isEmpty()) return -1.0;
+
+            Optional<Integer> result = Models.Horse.calculateNextLevelSeconds();
+
+            return result.isPresent() ? Math.ceil(result.get() / 60.0 * 100.0) / 100.0 : -1.0;
```

Comment:
I think you should just return the seconds here, then other functions can be used to display it however the user wants